### DISCUSSION TITLE
UI/UX Collaboration

### DISCUSSION THREAD

**Discussion by psanders** [fonoster/yaps#331] [Ideas]
👋  Hello community,

I am opening this discussion so we can start collaboration around the project's UI/UX. 

Initially, we want to brainstorm ideas for the WebUI and then make them into small chunks of collaborative work (as Github issues).

This could be an excellent opportunity for designers who want to get into Open Source, including juniors building their portfolios.

What do you all think?

Link to Figma: https://www.figma.com/file/6bDNA0jLJ9rjpTydSkIa33/Fonoster-Design?node-id=0%3A1

**Comment by theewiz** [comment]
🚀 Great initiative @psanders! This will give Product Designers (especially junior Designers) an opportunity to collaborate with developers, and bounce off ideas.

Let's go🎉

**Comment by ojosamuel** [comment]
Looking forward to getting my hands on something. This is awesome

**Reply by psanders** [reply]
Thanks @ojosamuel 

If we can get something going, it could help the community.

**Comment by joyracheal** [comment]
this will be indeed an opportunity to work with developers and also brainstorm with fellow designs to build a great product

**Reply by psanders** [reply]
@joyracheal,

This is starting to look like a squad 🔥

**Comment by OSETIEMORIA** [comment]
This is definitely going to be an awesome way to collaborate and learn from other designers. Looking forward to building and being a part of the squad!

**Comment by psanders** [comment]
@OSETIEMORIA @joyracheal @ojosamuel @theewiz 👋

I am looking at Thursday at 9 AM EST for a quick intro meeting. Who is up for this?

**Reply by OSETIEMORIA** [reply]
Yes me! I’m up for this.

**Comment by theewiz** [comment]
@psanders sure that's cool. Let's meet

**Comment by joyracheal** [comment]
I'm certainly up 
woops

**Comment by Victorybrown** [comment]
Hi, @psanders This is great! 
I'm currently sourcing for open source projects to get designers to contribute to. I would love to watch this discussion to know when it'll commence and we can begin walking towards it. If you'd love to have a conversation about this, I'd love to.

**Reply by psanders** [reply]
I will be happy to have a coffee chat with you.

**Comment by hafsalm** [comment]
@psanders, Having dedicated the past 11 years to practicing design, I would sincerely appreciate the opportunity to contribute to this inspiring project 👍
